src/pooled_connection/mod.rs

```diff
@@ -293,6 +293,7 @@ where
     }
 }
 
+#[allow(dead_code)]
```

Comment:
Haven't touched this part of the code but CI for rust beta doesn't pass w/o this. LMK if you want me to revert